src/default_client.rs

```diff
@@ -742,60 +1256,162 @@ impl<A: ClientAdapter + Sync + Send> Client for DefaultClient<A> {
         self.send(request).await
     }
 
+    /// List messages from a chat channel.
+    ///
+    /// The chat channel id can be retrieved by using [`Socket::join_chat`].
+    ///
+    /// TODO: How does forward work?
```

Comment:
It looks like this endpoint is also missing in the official docs

Reply:
I am not sure what you mean by "forward". 

The list chat messages history docs can be found here: https://heroiclabs.com/docs/social-realtime-chat/#message-history

Reply:
Ah thanks. However, the documentation is not describing the use of `forward`.

Reply:
Oh I see what you're saying. I believe it refers to the direction of the pagination (e.g., are you looking backwards or forwards for further records.) @zyro @seanpackham @gpene